app/src/main/java/com/nutomic/syncthingandroid/service/NotificationHandler.java

```diff
@@ -171,6 +172,19 @@ public void showEventNotification(String text, PendingIntent pi, int id) {
         mNotificationManager.notify(id, n);
     }
 
+    public void showStoragePermissionRevokedNotification() {
+        Intent intent = new Intent(mContext, FirstStartActivity.class);
+        Notification n = getNotificationBuilder(mInfoChannel)
+                .setContentTitle(mContext.getString(R.string.syncthing_terminated))
+                .setContentText(mContext.getString(R.string.toast_write_storage_permission_required))
```

Comment:
Don't see the string anywhere?

Reply:
Its been reused from the Message that explains the storage permission in FirstStartActivity